### PR TITLE
cart page created successfully

### PR DESCRIPTION
**Pull request by ivinayakg**
Web capture_12-2-2022_18931_127 0 0 1
Web capture_12-2-2022_18921_127 0 0 1

### CODE REVIEW

File: components/cartTotal/index.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  padding: 1.2rem;
+}
+.cart_total > * {
+  width: 100%;
+  border-bottom: 1.5px solid rgba(0, 0, 0, 0.2);
+  padding-bottom: 8px;
+  margin-top: 8px;
+}
+
+.cart_totalDetails {
```
Comment (VLeads) [inline components/cartTotal/index.css:18]: it  is suggested to write lassnames only in small letters

**Review by VLeads — APPROVED** [review]
Added a suggestion. Else, code is clean.

File: components/cartProduct/index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<link rel="stylesheet" href="./index.css" />
+<link rel="stylesheet" href="/css/utility.css">
+
+<div class="container">
+  <div class="cartProduct cartProduct--portrait">
+    <div class="cartProduct_img">
+      <img src="./assets/productImage.png" alt="" />
```
Comment (shubhmehra) [inline components/cartProduct/index.html:7]: please use valid alt names for better accessibility

Comment (shubhmehra) [reply components/cartProduct/index.html:7]: same change wherever required

File: screens/cart/index.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
+.hero .container {
+  width: 100%;
```
Comment (narharikale) [inline screens/cart/index.css:2]: can we make a utility class for `width =100%`  ?  so we don't  have to write a class name for just one property

File: components/cartTotal/index.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  border: none;
+}
+
+.cart_totalAction button {
+  transition: none;
+  width: 100%;
+  margin-top: 8px;
+}
+
+@import url(/css/utility.css);
```
Comment (narharikale) [inline components/cartTotal/index.css:46]: is there any specific reason to import CSS at the end?

File: components/cartProduct/index.html (hunk 2 above)
Comment (ivinayakg) [reply components/cartProduct/index.html:7]: ok will try to do better

File: screens/cart/index.css (hunk 3 above)
Comment (ivinayakg) [reply screens/cart/index.css:2]: i was thinking the same thing when I created this.

File: components/cartTotal/index.css (hunk 4 above)
Comment (ivinayakg) [reply components/cartTotal/index.css:46]: yeah, to make sure that my utility classes can overwrite any css.

File: components/cartProduct/index.css [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+.cartProduct--portrait {
+  height: 13rem;
+}
+
+.cartProduct_img {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```
[the remark this answers is not in the document]
Comment (ivinayakg) [reply components/cartProduct/index.css:20]: have done that, but am using this here to make sure a component doesn't depend on external styling for its main styling

**Pull request merged**